pt/news/_posts/2016-10-20-fukuoka-ruby-award-2017.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: news_post
+title: "2017 Fukuoka Ruby Award Competition - Entradas serão avaliadas pelo Matz"
+author: "Fukuoka Ruby"
+translator: "Filipe Rocha"
+date: 2016-10-20 00:00:00 +0000
+lang: pt
+---
+
+Caros entusiastas de Ruby,
+
+O Governo de Fukuoka, Japão, juntamente com "Matz" Matsumoto gostaria de
+vos convidá-lo a entrar no seguinte concurso Ruby. Se já desenvolveu um
```

Comment:
Brazilian review, I'm not sure but here in Brazil we would say: `gostaria de convidá-los` instead of `vos convidá-lo`, just ignore if this is correct in Portugal 😄